### DISCUSSION TITLE
Monthly ESMValTool meeting March 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#18] [Meetings]
The next monthly ESMValTool meeting will be in March. The result of the poll is that the meeting will take place on

Thursday, March 10 at 15:00 CET (14:00 GMT).

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

@ESMValGroup/esmvaltool-developmentteam

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we can decide to talk about specific topics with the entire group or make smaller breakout groups to continue the discussion there.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the February meeting can be found here: #9

**Comment by valeriupredoi** [comment]
cheers @bouweandela - as usual, am flexible but no 8 or 9am's for me please :grin:

**Comment by rswamina** [comment]
We have two discussion points from the @ESMValGroup/userengagementteam meeting in February:

1. How to keep track of users of ESMValTool (including projects and groups): We want to collect information on ways to do this and incorporate those ways. @rswamina will provide an initial set of suggestions for discussion from the user engagement team.
2. We thought to have a central location to show the list of participating institutions/organizations for ESMValTool and links to that page from elsewhere. This is to ensure we have an updated list at all times and one that requires updates just at one location. @remi-kazeroni will bring this up for discussion.

**Comment by katjaweigel** [comment]
A topic that came up in PR ESMValGroup/ESMValTool#2156 is, if/how we can split large recipes (e.g. recipe_flato13ipcc.ym) into smaller parts, to make it easier to run/test them.

**Reply by zklaus** [reply]
We briefly discussed this at the monthly meeting and decided that this should be tackled in two parts.

One is strictly what @katjaweigel is asking about, that is a technical means to split up recipes, possibly including something like a meta recipes that allows binding together smaller recipes. @katjaweigel, could you open an issue to discuss and flesh out these ideas?

The other is more of a set of guidelines on how to layout recipes, how long to make them, whether to mix for example observational cmorizers and diagnostics in the same PR or to better keep them separate. For this, @hb326 agree to open a discussion.

**Reply by katjaweigel** [reply]
Sorry, that I could not attend the meeting last week. I started an issue about the here: https://github.com/ESMValGroup/ESMValTool/issues/2598

**Comment by bsolino** [comment]
It seems the Doodle link is not working for me, I can only see an empty calendar and a message saying "All slots have been booked".

**Reply by zklaus** [reply]
Same for me.

**Reply by bsolino** [reply]
It seems Doodle is experiencing issues: https://uptime.doodle.com/

**Reply by bouweandela** [reply]
It looks like Doodle is back online. Please fill the doodle today: I will select a time by the end of the day.

**Comment by bouweandela** [comment]
@ESMValGroup/esmvaltool-developmentteam Thank you to everyone who filled the poll. The meeting will take place on Thursday, March 10 at 15:00 CET (14:00 GMT). Hope to see you there!

**Comment by valeriupredoi** [comment]
aloha! Zoom link says meeting hasn't started yet, waddup?

**Comment by valeriupredoi** [comment]
**Minutes**

**Attendance**: Klaus, Remi, Birgit, V, Saskia, Rob King, Ranjini, Manu, Brei, Katherine, Emma, Bouwe from the garden for 5min, Greg Munday
**Introduction round**: peeps telling us what they been up to lately
**Discussion points**:
- Ranjini and the User Engagement stuff: UET meeting once a month, maintain the Tutorial, tutorials in person and online, work on documantation (readibility) and first point of entry for the newbies;
  - **point at hand**: who is actually using the Tool - establish some sort of accounting (non-intrusively) -- look at the number of forks maybe, but asking what's the best way to start a discussion on this? Klaus: Discussions item is a good idea; Birgit: in-person discussion at wkshps ideal; --> Ranjini will open an item in the general Discussions. WRT wkshps Klaus mentions we should get more of the scientists
  - Remi for Ranjini (tutorial): successful tutorial at DLR (18 participants!) using Mistral
  - coalesce lists of contributors/developers (bunch of lists all over the shop) - define a central list that gets updated at once - Klaus mentioned `cff` checks through to CITATIONS; maybe best to open an issue he says; when moving the website to GH Pages also a good chance to centralize
- technical issue to be opened to split large recipes (Katja's point about `flato` recipe; Katja not here tho, so not much debating about it); Birgit not very happy with large recipes either (who would be?) - write up guidelines about how to split recipes; Klaus trying to explain why some recipes are large: reproduce paper/IPCC report figures - many models etc, so there are two approaches: technical and conceptual - to be discussed in an issue, Birgit is nice enough to start the ball rolling on the discussion for guidelines (she be opening a Discussions item)
- Emma: ask about the OBS auto download stuff - new cmorizer interface: https://github.com/ESMValGroup/ESMValTool/discussions/2560
- Questions: scripts are only Tier 1 and 2, what about Tier 3? Remi answers: T3 is good too, need to store credentials etc, instructions are in the Documentation.
Question: MO needs to grab OBS data so they can run ESMValTool, for this purpose, is there any way to log the downloads (user, date, amount downloaded etc) so they can have an inventory? Remi answers: it's a bit of an on-the-fly situation - if data gets uploaded, we sync the repos at DKRZ and JASMIN, no cron job that monitors if data can be added, let's open an issue for better syncing mechanisms. Question from Ranjini: how often DKRZ syncs to JASMIN? About when Remi is in a good mood, or when there's incoming data to DKRZ. Also Ranjini: possible datasets available only on certain machines (example only at MO). Remi: lots of good points, supporting Klaus idea of a possible workshop on this topic (obv more discussion needed)
- Birgit: workshoppe!! Birgit asked about an in-person meeting in May/June at DLR - ATM no, but regulations may change from the Bundestag on 20 March - not clear yet. It's gonna be hybrid either way, location TBD given the insecurity of German regulations - somewhere else where no regs? Ranjini: gotta be hybrid; Emma: suggesting a workshop at the MO in October/November - cool!
Birgit: we should find a date if wkshp in June

**Comment by valeriupredoi** [comment]
@hb326 place holder for a Doddlepoodle about the early summer workshoppe :beer: :de: Discussion item https://github.com/ESMValGroup/ESMValTool/discussions/2588

**Comment by valeriupredoi** [comment]
@bouweandela et al, I noticed we don't have a repo for the Monthly meetings minutes in https://github.com/ESMValGroup/Community - should I create one and plop today's meetings as a start?

**Reply by bouweandela** [reply]
I always add the meeting notes to the discussion, in case people like to discuss more online after the actual meeting. Maybe we could add a note about where to find these notes to the README.md? Would that work for you?

**Comment by bouweandela** [comment]
Thanks for helping out with organizing the meeting today!
